ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -177,6 +177,10 @@ def get_resources(dict_configs, cdump='gdas'):
             strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_BATCH;">\n')
         if scheduler in ['slurm'] and task in ['getic', 'arch']:
             strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_SERVICE;">\n')
+        if scheduler in ['slurm'] and machine in ['JET'] and task in ['fcst']:
+            strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_BATCH;">\n')
+        if scheduler in ['slurm'] and machine in ['JET'] and task in ['post']:
+            strings.append(f'\t<!ENTITY PARTITION_{taskstr} "&PARTITION_BATCH;">\n')
```

Comment:
You don't need these lines if you change line 176 from:

```
if scheduler in ['slurm'] and machine in ['ORION'] and task not in ['getic', 'arch']:
```

to

```
if scheduler in ['slurm'] and machine in ['ORION',  'JET'] and task not in ['getic', 'arch']:
```

Reply:
Will create different XML.
Recommend to keep the version to reproduce the control.

Reply:
Different XML's are fine.  I would like to see the XML and the differences (if any) resulting in the execution.

Reply:
Modified as requested.